rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deltalake"
-version = "0.4.0"
+version = "0.4.11"
```

Comment:
curious what's the reason for going from 0 to 11 here?

Reply:
Sorry... still testing the env var in GitHub actions... will revert it back to 0.4.0 before finalizing